client/config/index.js

```diff
@@ -0,0 +1,11 @@
+let config;
+
+if ( typeof window !== 'undefined' && window.config ) {
+	config = window.config;
+	config.anyEnabled = window.anyEnabled;
+	config.isEnabled = window.isEnabled;
+} else {
+	throw new Error( 'config not available' );
+}
```

Comment:
absolutely - I didn't realize/think about that it was a function. that's too bad, but thanks for verifying that and adjusting accordingly